src/main/resources/templates/clojure-lsp/clientSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "caseSensitive": true,
   "completion": {
-    "useContextAwareSorting": true
+    "useContextAwareSorting": true,
+    "useTemplateForInvocationOnlyCodeSnippet": false
```

Comment:
As before, disabling editor templates for invocation-only code snippets in all bundled templates so that newly-created language server definitions will have this (IMO preferred) behavior by defafult.